.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- First time contributors: Take a moment to review CONTRIBUTING.md! -->
+<!-- Remove sections if not applicable -->
+## Summary of changes
+
+<!-- Summary goes here -->
+
+Closes <!-- issue number here -->
```

Comment:
I'd make this a checkbox in the checklist section.  Fine either way.

Reply:
Yeah, I debated that. The reason I decided against it were:  1. that it's not really an "action item", it's a piece of information to convey in the PR and 2. I wasn't sure that the github auto-closing checker would recognize it in a checklist.

Reply:
I think the latter is probably not a real concern, but I'll leave it as is unless it's confusing people.

Reply:
Fair enough, not a strong preference.  FWIW the pandas template puts the "closes" in the checklist.